installer/PowerToysSetup/Product.wxs

```diff
@@ -1000,15 +1000,19 @@
       <?endforeach?>
       <!-- !Warning! Make sure to change Component Guid if you update something here -->
       <Component Id="Module_PowerRename" Guid="40D43079-240E-402D-8CE8-571BFFA71175" Win64="yes">
-        <RegistryKey Root="HKLM" Key="Software\Classes\CLSID\{0440049F-D1DC-4E46-B27B-98393D79486B}">
+        <RegistryKey Root="HKCU" Key="Software\Classes\CLSID\{0440049F-D1DC-4E46-B27B-98393D79486B}">
           <RegistryValue Type="string" Value="PowerRename Shell Extension" />
           <RegistryValue Type="string" Name="ContextMenuOptIn" Value="" />
           <RegistryValue Type="string" Key="InprocServer32" Value="[PowerRenameInstallFolder]PowerToys.PowerRenameExt.dll" />
           <RegistryValue Type="string" Key="InprocServer32" Name="ThreadingModel" Value="Apartment" />
         </RegistryKey>
-        <RegistryKey Root="HKLM" Key="Software\Classes\AllFileSystemObjects\ShellEx\ContextMenuHandlers\PowerRenameExt">
+        <RegistryKey Root="HKCU" Key="SOFTWARE\Classes\Directory\shellex\ContextMenuHandlers\PowerRenameExt">
           <RegistryValue Type="string" Value="{0440049F-D1DC-4E46-B27B-98393D79486B}"/>
         </RegistryKey>
+        <RegistryValue Root="HKCU"
+                       Key="SOFTWARE\Classes\*\ShellEx\ContextMenuHandlers\PowerRenameExt"
+                       Value="{0440049F-D1DC-4E46-B27B-98393D79486B}"
+                       Type="string" />
```

Comment:
Was this one missing before? :O

Reply:
nah, this was a leftover of trying to make PowerRename work when right-click inside folder (not items) bit it didn't work. moving back to the same reg key...

Reply:
works the same both ways, though